packages/fxa-auth-server/lib/senders/templates/subscriptionAccountFinishSetup.txt

```diff
@@ -4,7 +4,7 @@
 
 {{{t "Your payment is processing and may take up to four business days to complete. Your subscription will renew automatically each billing period unless you choose to cancel." }}}
 
-{{#if productPlan }}{{ productPlan }}{{/if}}
+{{#if productName }}{{ productName }}{{/if}}
```

Comment:
Is this just a mistake in the old txt file, `productPlan` is never passed in?

Reply:
I think so since the HTML had the product name within the Payment Details section and the plain text did not as the variable used is `productPlan` (see below).

<img width="636" alt="Screen Shot 2021-12-20 at 12 35 30 PM" src="https://user-images.githubusercontent.com/28129806/146808724-e6f6c4fd-2820-4ce5-b905-99089d3505eb.png">

<img width="690" alt="Screen Shot 2021-12-20 at 12 34 18 PM" src="https://user-images.githubusercontent.com/28129806/146808635-adce6d8a-9d32-4475-8c14-beb5c3b43ae7.png">

Reply:
I took another look at this and it's odd that the HTML would call out if `productName` was undefined since `productName` is required for the email. 🤔 I'll add this to the Confirmation Needed doc.

Reply:
I just checked [this function in email.js](https://github.com/mozilla/fxa/blob/main/packages/fxa-auth-server/lib/senders/email.js#L1995) and we never pull in `productPlan`. We probably forgot to update it at some point. It's probably good to note in that doc if even just for QA to see that it was supposed to be there before and wasn't but is present in the new template. Good catch 😎